### PR TITLE
e2e: new tests are automated

### PR DESCRIPTION
**Pull request by pavloburykh**
Added following checks: 
https://ethstatus.testrail.net/index.php?/cases/view/702132
https://ethstatus.testrail.net/index.php?/cases/view/702135
https://ethstatus.testrail.net/index.php?/cases/view/702136
https://ethstatus.testrail.net/index.php?/cases/view/702344

### CODE REVIEW

File: test/appium/tests/medium/test_profile_m.py [hunk 1]
```diff
@@ -200,6 +200,68 @@ def test_profile_chat_two_minutes_gap_in_public_and_no_gap_in_1_1_chat_and_group
             chat_view.back_button.click()
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(702367)
+    def test_chat_mute_unmute_contact(self):
```
Comment (churik) [inline test/appium/tests/medium/test_profile_m.py]: add push in test name

File: test/appium/tests/medium/test_profile_m.py [hunk 2]
```diff
@@ -200,6 +200,68 @@ def test_profile_chat_two_minutes_gap_in_public_and_no_gap_in_1_1_chat_and_group
...
+        self.device_1.get_app_from_background()
+
+        chat_1.just_fyi('Verify that message from unmuted user is actually received')
+        if not chat_1.chat_element_by_text(message_after_unmute).is_element_displayed():
+            self.errors.append("Message from unmuted contact hasn't been received")
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702368)
+    def test_chat_group_chat_reopen_app_on_last_viewed_chat(self):
```
Comment (churik) [inline test/appium/tests/medium/test_profile_m.py]: test is not related to group_chat, better to remove it from name

File: test/appium/views/chat_view.py [hunk 3]
```diff
@@ -114,6 +114,23 @@ def click(self):
...
+    def click(self):
+        self.scroll_to_element()
+        self.wait_for_element().click()
+
+
+class ProfileUnmuteContactButton(Button):
+    def __init__(self, driver):
+        super(ProfileUnmuteContactButton, self).__init__(driver, accessibility_id="Unmute-item-button")
+
+    def click(self):
```
Comment (churik) [inline test/appium/views/chat_view.py]: not sure if we need to define separate class for re-defining only this method, as `scroll_and_click` method exists

**Review by churik — APPROVED** [review]

File: test/appium/views/chat_view.py (hunk 3 above)
Comment (pavloburykh) [reply test/appium/views/chat_view.py]: thanx @churik, refactored

File: test/appium/tests/medium/test_profile_m.py (hunk 2 above)
Comment (pavloburykh) [reply test/appium/tests/medium/test_profile_m.py]: thanx @churik, renamed

File: test/appium/tests/medium/test_profile_m.py (hunk 1 above)
Comment (pavloburykh) [reply test/appium/tests/medium/test_profile_m.py]: thanx @churik, renamed

**Pull request merged**